### ISSUE TITLE
Implement Adam optimizer & Tversky loss in train model function

### ISSUE THREAD

**Issue by Jacob-Meert** [psu-nittanyaiadvance/26-S-Lockheed-2#24]
### Description of the Task:

Utilize the milesial/Pytorch-UNet repository as example in how to implement our selected loss function (Tversky loss) and optimizer (Adam) prior to the training loop in train.py. This code should be in the train_model function.

### Acceptance Criteria:

- Code should create the optimizer utilized in the training loop
- Code should create the loss function utilized in the training loop
- Each of these implementations should be supported by theoretical understanding.
- Parameters required by these objects should be added to the function definition.
- Code should follow the style of milesial/Pytorch-UNet repository.
- Code should be entirely understood.
- Code should take into account the larger picture of the function it is in, and account for it in its design.
